airbyte-integrations/bases/base-normalization/setup.py

```diff
@@ -31,7 +31,7 @@
     author_email="contact@airbyte.io",
     url="https://github.com/airbytehq/airbyte",
     packages=setuptools.find_packages(),
-    install_requires=["airbyte-protocol", "dbt==0.18.1"],
+    install_requires=["airbyte-protocol", "dbt>=0.18.1", "pyyaml"],
```

Comment:
What's the benefit of allowing > versions of dbt? Shouldn't we bump the version to upgrade?